src/app/zap-templates/zcl/data-model/chip/concentration-measurement-cluster.xml

```diff
@@ -0,0 +1,1028 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <cluster>
+    <name>Carbon Monoxide Measurement</name>
+    <domain>Measurement &amp; Sensing</domain>
+    <description>Attributes for reporting carbon monoxide concentration measurements</description>
+    <code>0x040C</code>
+    <define>CARBON_MONOXIDE_MEASUREMENT_CLUSTER</define>
+
+    <client tick="false" init="false">true</client>
+    <server tick="false" init="false">true</server>
+
+    <attribute side="server" code="0x0000" define="MEASURED_VALUE" type="single" min="0" max="0xFFFFFFFF" writable="false" isNullable="true" optional="true">MeasuredValue</attribute>
+    <attribute side="server" code="0x0001" define="MIN_MEASURED_VALUE" type="single" min="0" max="0xFFFFFFFF" writable="false" isNullable="true" optional="true">MinMeasuredValue</attribute>
+    <attribute side="server" code="0x0002" define="MAX_MEASURED_VALUE" type="single" min="0" max="0xFFFFFFFF" writable="false" isNullable="true" optional="true">MaxMeasuredValue</attribute>
+    <attribute side="server" code="0x0003" define="PEAK_MEASURED_VALUE" type="single" min="0" max="0xFFFFFFFF" writable="false" isNullable="true" optional="true">PeakMeasuredValue</attribute>
+    <attribute side="server" code="0x0004" define="PEAK_MEASURED_VALUE_WINDOW" type="int32u" min="0" max="259200" writable="false" default="1" optional="true">PeakMeasuredValueWindow</attribute>
```

Comment:
Spec has uint8 for the type, no?  And a different max?  I assume this is based on some un-merged spec PR, but this PR does not say which one, which makes reviewing a bit painful.

Reply:
You are absolutely right! This got lost a bit since the PR which changed this was a kind of omnibus PR im really sorry! I added a reference to the PR in question to the PR description at the top 🔝